iroh-net/src/magicsock.rs

```diff
@@ -286,18 +288,18 @@ impl MagicSock {
 
     /// Returns `true` if we have at least one candidate address where we can send packets to.
     pub(crate) fn has_send_address(&self, node_key: PublicKey) -> bool {
-        self.connection_info(node_key)
+        self.node_info(node_key)
             .map(|info| info.has_send_address())
             .unwrap_or(false)
     }
 
-    /// Retrieve connection information about nodes in the network.
-    pub(crate) fn connection_infos(&self) -> Vec<ConnectionInfo> {
+    /// Return all the [`NodeInfo`]s of all nodes in the node map.
+    pub(crate) fn node_info_all_nodes(&self) -> Vec<NodeInfo> {
         self.node_map.node_infos(Instant::now())
```

Comment:
it's a bit confusing that we use three names now for this

- `node_infos` in the `node_map`
- `node_info_all_nodes` here
- and `node_info_iter`

Reply:
What about unifying to `all_node_infos`? I assume `node_infos` is too close to `node_info` to be a memorable method name?

Then I wouldn't mind `all_node_infos_iter` as the one exception